maintainers/maintainer-list.nix

```diff
@@ -8461,6 +8461,11 @@
     githubId = 60962839;
     name = "Mazen Zahr";
   };
+  GKHWB = {
+    github = "GKHWB";
+    githubId = 68881230;
+    name = "GKHWB";
+  };
```

Comment:
Suggestion, not necessary: When we are at it, would you be OK with adding some additional means of contacting you? Matrix, email? It is easier when it is needed than GitHub.